test/models/code_school_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class CodeSchoolTest < ActiveSupport::TestCase
+  def setup 
```

Comment:
Since the `code_school.rb` model file does not have any methods in it yet, the only thing you can really test for is validations.  So until we have some methods in there, you can probably just delete this file.